### ISSUE TITLE
Add info about other projects to the NUnit Readme.md

### ISSUE THREAD

**Issue by CharliePoole** [nunit/nunit#1538]
See issue nunit/nunit3-vs-adapter#174

We should have a section of the readme that lists other nunit projects and points to the repos. This may allay confusion on the part of users.

**Comment by ChrisMaddock** [comment]
I wonder if NUnit should introduce a standard readme for all the repos?

Was thinking before about making sure the docs were always linked from the readme. If other projects should be linked too (which I agree with!) maybe we could come up with a standard format?

Something like: 

- Project Blurb
- Docs link
- Licence link
- Contributing-How-To
- Links to other projects

**Comment by rprouse** [comment]
@ChrisMaddock that sounds like a good idea. To your list, probably right after the docs link, I would like to see a Quick Start section with just enough info to get people up and running and then send them to the docs for the full info.

**Comment by CharliePoole** [comment]
@ChrisMaddock We can try to make this one so it can serve as a template. That said, I think we will end up with a "standard-ish" rather than a standard readme layout. As I have tried to standardize other things across projects, I have encountered various unexpected differences due to the nature of the projects.

@rprouse I'd rather not have a quick start in two different places though. Would a link to the quick start page suit you?
